hookman/__main__.py

```diff
@@ -80,13 +80,13 @@ def package_plugin(specs_path: str, package_name: str, plugin_dir: str, dst_path
 
 @cli.command()
 @click.argument('specs_path', type=click.Path(exists=True))
-@click.argument('shared_lib_name')
+@click.argument('plugin_id')
```

Comment:
You could use `@click.argument('plugin_id', metavar='ID')` this way the function arg name will be `plugin_id` but the displayed value in the help message will be just `ID` (the same case is used) .

Reply:
In my opinion, is better to have "PLUGIN ID" instead of only "ID" for the arguments.
With `metavar`
```python
Usage: hookman generate-hook-specs-h [OPTIONS] SPECS_PATH ID

  Generates or update the hook_specs.h header file.

Options:
  --dst-path TEXT  Path to where the files will be written
  --help           Show this message and exit.
```

Without `metavar`
```python
Usage: hookman generate-hook-specs-h [OPTIONS] SPECS_PATH ID

  Generates or update the hook_specs.h header file.

Options:
  --dst-path TEXT  Path to where the files will be written
  --help           Show this message and exit.
```